include/msgpack/object.hpp

```diff
@@ -511,6 +511,15 @@ inline T* object::convert(T* v) const
     return v;
 }
 
+template <typename T>
+inline bool object::convert_if_not_nil(T& v) const {
```

Comment:
Style suggestion: In other methods, you put the brace on a separate line. Maybe here too.